test/test_datetimeWrapper.js

```diff
@@ -523,6 +532,26 @@ describe('sundial', function() {
         expect(variance < 10 && variance >= 0).to.be.true;
       });
     });
+    describe('dateDifference', function() {
+      it('returns the difference between two timestamps', function() {
+        var t1 = '2013-01-22T04:25:21.000Z';
+        var t2 = '2013-01-23T04:25:21.000Z';
+
+        var diffHrs = datetimeWrapper.dateDifference(t1,t2,'hours');
+        expect(diffHrs).to.equal(-24);
 
+      });
+      it('returns the difference between two timestamps allowing for negative difference', function() {
+        var t1 = '2000-01-22T04:25:21.000Z';
+        var t2 = '2013-11-22T04:25:21.000Z';
```

Comment:
Can you make these test cases a little more obvious re: the birthday use case? i.e., pretend t1 is a birthday and make t2 exactly a day short of it, so that we can be absolutely assured it rounds in the correct direction?